### DISCUSSION TITLE
Are others seeing this? "We encountered an internal error. Please try again."

### DISCUSSION THREAD

**Discussion by pretherford** [amzn/selling-partner-api-models#3496] [General]
Are other developers seeing this error when calling:  POST /fba/inbound/v0/plans ?

```json
{
    "errors": [
        {
            "code": "InternalFailure",
            "message": "We encountered an internal error. Please try again.",
            "details": ""
        }
    ]
}
```

This error is happening with multiple seller accounts and is seemingly random although more frequent in recent days. Sometimes the same request will succeed minutes after the first error. We have open cases with developer support and are not getting a resolution. The response from the API is the same when using Postman or our application code.

ref:
https://developer-docs.amazon.com/sp-api/docs/fulfillment-inbound-api-v0-reference#createinboundshipmentplan

**Comment by AshwinPitroda** [comment]
@pretherford I am facing same error right now.

**Comment by hellokids86** [comment]
Seeing this error as well

**Comment by hellokids86** [comment]
@pretherford did you find a solution to this? If so, please share.
